lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -431,10 +539,63 @@ sub add_batch_job {
     return $results;
 }
 
-
+$json_schemas{get_batch_job_result} = joi->object->strict->props(
+    batch_id => $Zonemaster::Backend::Validator::batch_id->required
+);
 sub get_batch_job_result {
     my ( $self, $batch_id ) = @_;
 
     return $self->{db}->get_batch_job_result($batch_id);
 }
+
+my $rpc_request = joi->object->props(
+    jsonrpc => joi->string->required,
+    id => joi->integer->min(0)->required,
+    method => joi->string->regex("[a-zA-Z0-9_-]*")->required
+);
+sub json_validate {
```

Comment:
Rename this to `jsonrpc_validate`?

Reply:
Fine, done in commit [2ab8271](https://github.com/dotse/zonemaster-backend/pull/399/commits/2ab82718ffef924397ad4cf13bb75ac82657953b)